android/app/src/main/java/org/fossasia/openevent/activities/SearchActivity.java

```diff
@@ -100,16 +88,27 @@ public boolean onCreateOptionsMenu(Menu menu) {
 
     @Override
     public boolean onQueryTextChange(String query) {
-        if (query != null) {
-            searchQuery(query);
+        String query1 = query.toLowerCase(Locale.getDefault());
+        String wildcardQuery = String.format("*%s*", query1);
+        searchViewModel.setSearchText(wildcardQuery);
+
+        if (!query.equals("")) {
```

Comment:
Can you explain the use of this non-empty check ?

Reply:
I did this as if the query is empty then the `handleIntent` function will be invoked

Reply:
@srv-twry Do I proceed with this ?